README.md

```diff
@@ -617,4 +618,4 @@ Unless required by applicable law or agreed to in writing, software
 distributed under the License is distributed on an "AS IS" BASIS,
 WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
-limitations under the License.
+limitations under the License.
```

Comment:
Please keep the line break at the end of the file

Reply:
is that the change? I used to see a little icon if the \n was the diff 🚫

Reply:
I checked out the branch locally to see the difference and it looks like that. I remember the icons, too, but it looks like they are gone.